src/main/java/com/adobe/aem/compgenerator/utils/CommonUtils.java

```diff
@@ -271,4 +293,68 @@ private static BufferedWriter getFileWriterFromTemplate(String path,
         }
         return updatedPropertiesList;
     }
+
+    /**
+     * Returns the jar file used to load class clazz, or defaultJar if clazz was not loaded from a
+     * jar.
+     */
+    public static JarFile jarForClass(Class<?> clazz, JarFile defaultJar) {
```

Comment:
Asked in the other file if we actually need this function.  If we do, it looks like we never use the `defaultJar` parameter